db/updateDb.js

```diff
@@ -1,6 +1,6 @@
 const common = require('../common.js');
```

Comment:
# Getting ERROR
### error on line 15 when downloading data
```
(node:26194) UnhandledPromiseRejectionWarning: TypeError: records is not iterable
    at getByCountry (/home/digitalvertices/cLab/visualisation-cas-covid-19/db/updateDb.js:15:24)
```

(record data is ```undefined```)
### data
 - remove db file
 - run db/CreateDb.js
 - npm start and wait for update